ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -7636,3 +7635,45 @@ def test_torch_instance_sign(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# bitwise right shift
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_right_shift",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
Can we test using valid dtypes instead? :)

Reply:
Apologies @ZiadAmerr. I missed it while resolving merge conflicts. Just pushed the changes,